README.md

```diff
@@ -1,185 +1,167 @@
-# IPFS is the Distributed Web
+# IPFS powers the Distributed Web
 
-> A peer-to-peer hypermedia protocol to make the web faster, safer, and more open.
+### A peer-to-peer hypermedia protocol to make the web faster, safer, and more open.
 
-[![Forum](https://img.shields.io/badge/Discourse-%23Discussion forums-blue.svg)](https://discuss.ipfs.io)
+[![Forum](https://img.shields.io/badge/Discourse-%23Discussion%20forums-blue.svg)](https://discuss.ipfs.io)
 [![Matrix](https://img.shields.io/badge/matrix-%23ipfs%3Amatrix.org-blue.svg?style=flat-square)](https://matrix.to/#/#ipfs:matrix.org)
 [![IRC](https://img.shields.io/badge/irc-%23ipfs-brightgreen.svg?style=flat-square)](https://webchat.freenode.net/?channels=ipfs)
 [![Discord](https://img.shields.io/discord/475789330380488707?color=blueviolet&label=discord&style=flat-square)](https://discord.gg/24fmuwR)
 [![Changelog #204](https://img.shields.io/badge/changelog-%23204-lightgrey.svg)](https://changelog.com/204)
 
-Welcome to IPFS! Why not [watch a video demo](https://www.youtube.com/watch?v=8CMxDNuuAiQ) to get started?
-
-**Please post questions and ideas at [https://discuss.ipfs.io](https://discuss.ipfs.io) .**
```

Comment:
We should funnel users with questions or support needs to Discourse. We should have very prominent links on this page, at the top, on where to get help. If not like this, perhaps as first item of the TL;DR...

Reply:
Was already in the TL;DR, but moved it to the top and boldfaced it.

Reply:
wops